functions/src/utils/venue.js

```diff
@@ -1,10 +1,10 @@
 const { VenueIdSchema } = require("../types/venue");
 
 const VENUE_ID_REGEX = /[a-z0-9_]{1,250}/;
+const INVALID_SLUG_CHARS_REGEX = /[^a-zA-Z0-9]/g;
```

Comment:
Is it worth moving any slug logic into a `utils/slugs.ts` module?

Reply:
Not by my account https://github.com/sparkletown/sparkle/pull/2509#discussion_r740245059